src/components/Content/common/grid.tsx

```diff
@@ -2,7 +2,7 @@
 import { css } from '@emotion/react';
 import styled from '@emotion/styled';
 import { ReactNode } from 'react';
-import { Grid } from 'semantic-ui-react';
+import { Grid, SemanticWIDTHS } from 'semantic-ui-react';
```

Comment:
`SemanticWIDTHS`?

Reply:
this looks really odd, just making sure it's not a typo

Reply:
It's what vscode pulled out of the semantic types... 😛